pom.xml

```diff
@@ -43,31 +43,18 @@
   </scm>
   <properties>
     <annotation-indexer.version>1.12</annotation-indexer.version>
-    <branch-api.version>2.0.21</branch-api.version>
-    <git-plugin.version>3.9.1</git-plugin.version>
     <github-branch-source.version>2.4.1</github-branch-source.version>
-    <groovy.eclipse.batch.version>2.4.3-01</groovy.eclipse.batch.version>
-    <groovy.eclipse.compiler.version>2.9.2-01</groovy.eclipse.compiler.version>
-    <groovy.version>2.4.19</groovy.version>
-    <jacoco-maven-plugin.version>0.7.9</jacoco-maven-plugin.version>
+    <jacoco-maven-plugin.version>0.8.6</jacoco-maven-plugin.version>
     <java.level>8</java.level>
     <java.version>1.8</java.version>
-    <jenkins.version>2.164.1</jenkins.version>
-    <junit.version>4.12</junit.version>
-    <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+    <jenkins.version>2.204.1</jenkins.version>
+    <junit.version>4.13</junit.version>
+    <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
```

Comment:
Still need junit4?

Reply:
Yeah, that allows the JUnit 4 tests from the plugin injected tests to work